### ISSUE TITLE
Missed tile replace not work

### ISSUE THREAD

**Issue by CosmaVito** [ghybs/Leaflet.TileLayer.Fallback#8]
Hi,

i have a map in my web app (an angular 2 web app).
I have installed _Leaflet 1.0.1_ and then the latest version of _Leaflet.TileLayer.Fallback_

**I want to replace the missed tile with a static tile placed in a Content directory** (the name of thi tile is no-tile.png).

I set bounds for my map. But i think that is not related to this.
  
```
var openstreetmapLayer = new L.tileLayer.fallback("/Content/openstreetmap/{z}/{x}/{y}.png", {errorTileUrl: '/Content/no-tile.png' });

       
 let bounds = new L.LatLngBounds(new L.LatLng(38.83328935686689, 9.9700927734375), new L.LatLng(42.31184652369512, 8.000793457031264));
        this.map = new L.Map(this.el.nativeElement, {
            center: bounds.getCenter(),
            minZoom: 8,
            maxZoom: 18,
            zoomControl:true,
            layers: [
                openstreetmapLayer
            ],
            maxBounds: bounds,
            maxBoundsViscosity: 1.0
        });


        let latlngs = L.rectangle(bounds).getLatLngs();
        L.polyline(latlngs[0].concat(latlngs[0][0])).addTo(this.map);
```
But i have still this error

![immagine](https://user-images.githubusercontent.com/14861632/34389834-4262ead2-eb3c-11e7-9966-17650a0fafed.png)

Thanks

**Comment by ghybs** [comment]
Hi,

If you could provide a Plunker or JSBin that reproduces your issue, that would tremendously help in understanding your situation.

Feel also free to ask for help on Stack Overflow.

**Comment by ghybs** [comment]
Most probably fixed by PR #10.

Please feel free to re-open with a reproduction example if you think it is a different issue.
